scenarioo-client/app/manage/generalSettings/generalSettings.controller.js

```diff
@@ -55,6 +57,12 @@ function GeneralSettingsController($scope, BranchesResource, ConfigService) {
         }
     }
 
+    function getEngineRunning () {
```

Comment:
is no getter, does not even return what is loaded here - should be called something like `loadSearchEngineRunning`.